Source/Pawnmorphs/Esoteria/DebugUtils/PMDebugActions.cs

```diff
@@ -363,9 +363,17 @@ static void TryRevertTransformedPawn(Pawn pawn)
             if (pawn == null) return;
             var gComp = Find.World.GetComponent<PawnmorphGameComp>();
             (TransformedPawn pawn, TransformedStatus status)? tfPawn = gComp?.GetTransformedPawnContaining(pawn);
+
             TransformedPawn transformedPawn = tfPawn?.pawn;
+
             if (transformedPawn == null || tfPawn?.status != TransformedStatus.Transformed) return;
-            MutagenDef mut = transformedPawn.mutagenDef ?? MutagenDefOf.defaultMutagen;
+
+            MutagenDef mut = null;
+            if (transformedPawn is MergedPawns)
```

Comment:
TransformedPawn references should really hold a reference of the mutagen that transformed them. but this is fine for now